app/translations/es.json

```diff
@@ -171,7 +171,6 @@
   "hrmi.containers.Settings.hi.title": "NO UTILIZADO: País de ingresos altos",
   "hrmi.containers.Settings.hi.text": "NO UTILIZADO: Para estos países es mejor utilizar el estándar de evaluación de ingresos altos",
   "hrmi.containers.Settings.hi.textIndicator": "NO UTILIZADO: para estos países es mejor utilizar los indicadores empleados para el estándar de evaluación de ingresos altos",
-  "hrmi.containers.FAQs.title": "TODO",
```

Comment:
Rather than removing these I strongly suggest keeping them with the default value.

This will allow text editors  to add the translation(s) when they can and also ensure the same number of lines in each language file

Reply:
fixing this now